tools/go.mod

```
@@ -0,0 +1,3 @@
+module github.com/publicsuffix/list/tools
+
+go 1.21.4
```

Maybe we can afford to be a little bit more lax here? I found my local Go toolchain was a bit stale (1.20.x) and it refused to build:

> invalid go version '1.21.0': must match format 1.23

```suggestion
go 1.21
```